CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 1.71.8
+
+- Corrected CI/CD issue
+
+# 1.71.7
+
+- Corrected CI/CD issue
```

Comment:
Duplicate change here

Reply:
There is technically already a 1.71.7 published, the pipeline failed to publish the doc site last and as such I felt like I should repeat the description here. I also have a note to fix the pipeline for this validation to be caught earlier. Here is the link the merge to master that I am talking about: https://app.circleci.com/pipelines/github/smartprocure/futil-js/334/workflows/d9a6c488-59cf-4661-9ea6-3be38a5e4308/jobs/1921

Let me know if there is some other way to handle this in your mind.

Reply:
Updated last comment, will fix this.

Reply:
Duplicates fixed